commit 57d0db0e1dfa3fc25d0bfc146fec1c89a446a9ea
Author: Alan T. DeKok <aland@freeradius.org>
Date:   Wed Jan 22 15:11:33 2014 -0500

    Use size out output buffer, not size of output pointer

diff --git a/src/lib/print.c b/src/lib/print.c
index a6e5391..5bc5e02 100644
--- a/src/lib/print.c
+++ b/src/lib/print.c
@@ -266,7 +266,7 @@ size_t vp_prints_value(char *out, size_t outlen, VALUE_PAIR const *vp, int8_t qu
 			return strlen(out);
 		}
 
-		return fr_print_string(vp->vp_strvalue, vp->length, out, sizeof(out));
+		return fr_print_string(vp->vp_strvalue, vp->length, out, outlen);
 
 	case PW_TYPE_INTEGER:
 		if (vp->da->flags.has_tag) {
